cleverpush/src/main/java/com/cleverpush/CleverPush.java

```diff
@@ -3264,6 +3264,7 @@ public static void removeInstance() {
 
   public void setInitializeListener(InitializeListener initializeListener) {
     this.initializeListener = initializeListener;
+    fireInitializeListener();
```

Comment:
i mentioned to do this, but i am not sure if this creates any other issues, maybe marius is aware of anything